Description: Add copy of upstream license text
 The upstream author licenses their software following the Repository
 Integrated Licence Tracking Specification (RILTS). The license is
 stated in the commit messages, which contain the following stanza:
 .
 ©! I, Hugo Landau <hlandau@devever.net>, hereby licence these changes under the
 ©! licence with SHA256 hash
 ©! fd80a26fbb3f644af1fa994134446702932968519797227e07a1368dea80f0bc.
Origin: upstream, https://github.com/hlandau/rilts/blob/e4791634449efa0ae1a63046cbff583a1babb50f/licences/COPYING.MIT
Bug: https://github.com/hlandau/acme/issues/195
Last-Update: 2016-10-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- /dev/null
+++ b/LICENSE
@@ -0,0 +1,5 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
